<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Incomes</class>
 <widget class="QWidget" name="Incomes">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Поставки</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelItemDate">
       <property name="text">
        <string>Дата</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDateTimeEdit" name="dateTimeEdit">
       <property name="dateTime">
        <datetime>
         <hour>0</hour>
         <minute>0</minute>
         <second>0</second>
         <year>2019</year>
         <month>1</month>
         <day>1</day>
        </datetime>
       </property>
       <property name="date">
        <date>
         <year>2019</year>
         <month>1</month>
         <day>1</day>
        </date>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelItemMaterial">
       <property name="text">
        <string>Материал</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBoxMaterials"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelCount">
       <property name="text">
        <string>Количество</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinBoxItemCount">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelItemResponsible">
       <property name="text">
        <string>Ответственный</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="comboBoxResponsible"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelItemProducer">
       <property name="text">
        <string>Поставщик</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="comboBoxProducers"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="labelItemPosition">
       <property name="text">
        <string>Позиция</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="comboBoxPositions"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonItemInsert">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutEditing">
     <item>
      <widget class="QPushButton" name="pushButtonItemUpdate">
       <property name="text">
        <string>Обновить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonItemUpdateCancel">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
